### ISSUE TITLE
[BUG] DPLR cannot run in PLUGIN mode

### ISSUE THREAD

**Issue by ChunyiZh** [deepmodeling/deepmd-kit#1627]
### Bug summary

After the fix of the issue #1567 (https://github.com/deepmodeling/deepmd-kit/issues/1567), kspace style 'pppm/dplr' can be recognized now. As shown in the log file of lammps "Loading plugin: kspace pppm/dplr v2.0 by Han Wang". 

However, the DPLR simulation crashed after step 0 "ERROR: Non-numeric pressure - simulation unstable (src/fix_nh.cpp:1069)". The same simulation can run successfully with build-in mode LAMMPS. I think the plugin mode may has some bug for DPLR.

### DeePMD-kit Version

2.1.0

### TensorFlow Version

2.8.0

### How did you download the software?

Built from source

### Input Files, Running Commands, Error Log, etc.

Please find all the files attached.
[test1.zip](https://github.com/deepmodeling/deepmd-kit/files/8468700/test1.zip)

### Steps to Reproduce

Run job.sh file.

**Comment by njzjz** [comment]
I failed to reproduce the error.
<details>
<summary>log</summary>

```
   Step         Volume          Temp          PotEng         KinEng         TotEng         Press     
         0   116652.63      333.82991     -1899097.6      524.97387     -1898572.6     -1726.8304    
         1   116652.59      333.1781      -1899253.2      523.94885     -1898729.2     -429.91549    
         2   116652.52      332.8939      -1899252.7      523.50193     -1898729.2     -747.11954    
         3   116652.4       333.00884     -1899252.9      523.68268     -1898729.2     -985.74377    
         4   116652.24      333.38861     -1899253.5      524.27989     -1898729.2     -1072.4351    
         5   116652.01      333.85291     -1899254.3      525.01005     -1898729.3     -1023.7279    
         6   116651.73      334.23067     -1899254.9      525.60411     -1898729.3     -811.04943    
         7   116651.41      334.37484     -1899255.1      525.83082     -1898729.3     -501.86796    
         8   116651.06      334.2502      -1899254.9      525.63482     -1898729.2      8.8919222    
         9   116650.7       333.92426     -1899254.4      525.12225     -1898729.2      497.31713    
        10   116650.38      333.52272     -1899253.7      524.49081     -1898729.2      1046.4859    
        11   116650.11      333.22013     -1899253.2      524.01496     -1898729.2      1545.1506    
        12   116649.93      333.18664     -1899253.2      523.96229     -1898729.2      1930.6122    
        13   116649.85      333.48406     -1899253.6      524.43        -1898729.2      2148.5353    
        14   116649.89      333.9549      -1899254.4      525.17043     -1898729.2      2205.3647    
        15   116650.05      334.29775     -1899255        525.70959     -1898729.3      2158.37      
        16   116650.33      334.30167     -1899255        525.71576     -1898729.3      1913.2115    
        17   116650.71      333.96386     -1899254.4      525.18453     -1898729.3      1534.8356    
        18   116651.18      333.40412     -1899253.5      524.30429     -1898729.2      1080.4053    
        19   116651.71      332.74697     -1899252.5      523.27088     -1898729.2      590.2516     
        20   116652.27      332.08324     -1899251.5      522.22711     -1898729.2      98.596983    
        21   116652.83      331.51704     -1899250.6      521.33671     -1898729.2     -354.23803    
        22   116653.37      331.21982     -1899250.1      520.8693      -1898729.2     -754.64481    
        23   116653.88      331.3629      -1899250.3      521.09431     -1898729.2     -1027.1368    
        24   116654.33      331.99439     -1899251.3      522.08738     -1898729.2     -1111.7787    
        25   116654.71      332.95314     -1899252.8      523.59509     -1898729.2     -1080.6493    
        26   116655.04      333.89575     -1899254.4      525.07742     -1898729.3     -874.32266    
        27   116655.32      334.50627     -1899255.3      526.03751     -1898729.3     -544.80653    
        28   116655.57      334.66754     -1899255.6      526.29112     -1898729.3     -93.982212    
        29   116655.81      334.53953     -1899255.4      526.08981     -1898729.3      417.91326    
        30   116656.08      334.46063     -1899255.2      525.96574     -1898729.2      927.76309    
        31   116656.4       334.72735     -1899255.6      526.38518     -1898729.2      1349.8906    
        32   116656.79      335.37416     -1899256.7      527.40234     -1898729.3      1673.5062    
        33   116657.27      336.1286      -1899257.9      528.58875     -1898729.3      1833.7402    
        34   116657.86      336.6764      -1899258.8      529.45021     -1898729.3      1826.1425    
        35   116658.54      336.9004      -1899259.1      529.80247     -1898729.3      1691.433     
        36   116659.31      336.88568     -1899259.1      529.77931     -1898729.3      1425.9097    
        37   116660.17      336.73468     -1899258.9      529.54186     -1898729.3      1041.4962    
        38   116661.08      336.41182     -1899258.4      529.03414     -1898729.4      645.42475    
        39   116662.03      335.84417     -1899257.5      528.14146     -1898729.4      176.94404    
        40   116662.98      335.05205     -1899256.2      526.89579     -1898729.3     -201.81072    
        41   116663.93      334.24118     -1899254.9      525.62064     -1898729.3     -556.52372    
        42   116664.84      333.70177     -1899254.1      524.77238     -1898729.3     -781.23915    
        43   116665.71      333.58215     -1899253.9      524.58426     -1898729.4     -918.16161    
        44   116666.54      333.79493     -1899254.3      524.91887     -1898729.4     -932.15091    
        45   116667.31      334.1531      -1899254.9      525.48213     -1898729.4     -812.73421    
        46   116668.03      334.4793      -1899255.5      525.9951      -1898729.5     -576.45839    
        47   116668.73      334.65817     -1899255.8      526.27638     -1898729.5     -226.04016    
        48   116669.41      334.67114     -1899255.8      526.29679     -1898729.5      177.89133    
        49   116670.1       334.56962     -1899255.6      526.13713     -1898729.5      621.93608    
        50   116670.83      334.42043     -1899255.4      525.90252     -1898729.5      1027.7377    
        51   116671.61      334.23937     -1899255.1      525.61778     -1898729.5      1358.6612    
        52   116672.47      333.97051     -1899254.7      525.19499     -1898729.5      1603.3525    
        53   116673.41      333.60941     -1899254.1      524.62712     -1898729.5      1736.0146    
        54   116674.45      333.29313     -1899253.6      524.12975     -1898729.5      1716.2003    
        55   116675.58      333.18911     -1899253.5      523.96617     -1898729.5      1582.7014    
        56   116676.8       333.29922     -1899253.7      524.13932     -1898729.6      1337.6406    
        57   116678.1       333.4036      -1899253.9      524.30348     -1898729.6      998.38886    
        58   116679.44      333.22607     -1899253.7      524.02429     -1898729.7      615.46397    
        59   116680.83      332.69346     -1899252.9      523.18672     -1898729.7      266.54887
```
</details>

**Comment by njzjz** [comment]
The log shows that v2.0.3 is actually used.
v2.0.3 has a bug which has been fixed by #1397.

**Comment by njzjz** [comment]
Fixed by #1637.
